package.js

```diff
@@ -1,5 +1,5 @@
 Package.describe({
-  name: 'natestrauser:publish-performant-counts',
+  name: 'luixal:publish-performant-counts',
```

Comment:
I wonder if changing the package name here is something that would prevent Nate from merging your pull request?

Reply:
Ups! My bad :(

Reply:
But I'm not sure about changing definitions with `let` for `var`... why?